input_generation/generateBatchInput.ts

```diff
@@ -20,13 +20,23 @@ if (path.extname(targetFile) != '.json') {
 }
 
 const configFile = JSON.parse(fs.readFileSync(hyparConfigPath))
-const parameters = configFile.inputs
+const parameters = configFile.input_schema.properties
+
+// TODO: Expand this to support multiple model import keys. Currently,
+// All model import keys will be set to the same model.
+var modelInputKeys = {}
+const inputModel = args[2]
+if (path.extname(inputModel) != '.json') {
+    console.error(`Input model must be a .json file. Currently: ${inputModel}`)
+    process.exit()
+}
+configFile.model_dependencies.map((md) => (modelInputKeys[md.name] = inputModel))
```

Comment:
Only do this if configFile.model_dependencies

Reply:
Done.